paasta_tools/tron_tools.py

```diff
@@ -775,6 +776,15 @@ def format_tron_action_dict(action_config: TronActionConfig, use_k8s: bool = Fal
         if action_config.get_team() is not None:
             result["labels"]["yelp.com/owner"] = action_config.get_team()
 
+        if (
+            action_config.get_iam_role_provider() == "aws"
+            and action_config.get_iam_role()
+        ):
+            result["annotations"]["iam.amazonaws.com/role"] = ""
```

Comment:
do we actually need this if the service account is already associated with the role?